pandas/tests/series/conftest.py

```diff
@@ -31,3 +31,12 @@ def object_series():
     s = tm.makeObjectSeries()
     s.name = "objects"
     return s
+
+
+@pytest.fixture(params=[None, lambda x: x])
```

Comment:
same as above, would be ok with just moving these fixtures to pandas/conftest.py (as they are the same)

Reply:
done.